sql/mob_spawn_points.sql

```diff
@@ -58242,11 +58247,11 @@ INSERT INTO `mob_spawn_points` VALUES (17658244,'Hannequet','Hannequet',10200,46
 INSERT INTO `mob_spawn_points` VALUES (17658245,'Hannequet','Hannequet',10200,466.629,19.987,-32.151,40);
 INSERT INTO `mob_spawn_points` VALUES (17658246,'Hannequet','Hannequet',10200,472.575,19.572,-39.897,70);
 INSERT INTO `mob_spawn_points` VALUES (17658247,'Ephemeral_Murex','Ephemeral Murex',10193,439.16,20.968,21.467,100);
-INSERT INTO `mob_spawn_points` VALUES (17658248,'Tejas','Tejas',10230,0,0,0,130);
-INSERT INTO `mob_spawn_points` VALUES (17658249,'Tejas','Tejas',10230,0,0,0,160);
-INSERT INTO `mob_spawn_points` VALUES (17658250,'Tejas','Tejas',10230,0,0,0,190);
-INSERT INTO `mob_spawn_points` VALUES (17658251,'Tejas','Tejas',10230,0,0,0,220);
-INSERT INTO `mob_spawn_points` VALUES (17658252,'Tejas','Tejas',10230,0,0,0,250);
+INSERT INTO `mob_spawn_points` VALUES (17658248,'Tejas','Tejas',0,-544,-3,114,12);
```

Comment:
Tejas  and a couple others actually have valid group IDs in darkstar, I just double checked to make sure other mobs didn't have same group ID also and he has 10230 all to himself in their file already.